CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+  The type of governance action. Note that the three possible treasury actions involve Lovelace amounts $T_0$, $T_1$, $T_2$, and $T_3$.
+
+* **Constitutional Committee**<br/>
+  A value of :heavy_check_mark: indicates that `Quorum-Many` Constitutional Committee 'yes' votes are required.<br/>
+  A value of :x: means that Constitutional Committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
+
+* **AVST**<br/>
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST action**<br/>
+  The consequence of the voting stake being below the AVST.
+  * **Required**<br/>
+    This governance action cannot be ratified without achieving the AVST threshold.
+  * **SPO**<br/>
+    For this governance action, SPO vote endorsement is not required if there is a sufficient active voting stake.
+    Otherwise, SPO vote endorsement is required.
+
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Action | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---         | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | Required    | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO         | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | Required    | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO         | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -           | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO         | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO         | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO         | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO         | $R_{6c}$ |
+
+Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
```

Comment:
Shouldn't the parameters here only be amendable by the constitution? Or break 5 out into update non-governance protocol parameter changes and governance protocol parameter changes, seems like there should be different voting thresholds for these two cases.

Reply:
In particular, if the thresholds are updateable protocol parameters, then the protocol parameter update action needs to be highly scrutinised since a malicious parameter update action can cause a lot of damage to the governance process.

Reply:
Indeed, this is why we have made no decisions about this yet. if the overall plan gets accepted, this is one of the details that requires a lot of scrutiny.

Reply:
I think it's not 100% clear what this CIP is supposed to be. It seems like it's more of a plan, and not a complete specification of the change. It certainly doesn't seem like the first version of the spec is going to be detailed enough to "enable alternative implementations" since it's going to be missing a lot of details. I guess maybe that's fine, maybe we just need to make it clear that the CIP is not complete yet.

Reply:
I don't think we've had a singe CIP yet which effected the ledger and was complete enough to enable alternative implementations, that's what the formal specifications have been for. But you are absolutely right, there are still a lot of details needed. It's a rather large and important change, and we wanted to engage in the conversation with the community sooner rather than later, to make sure that the overall idea had enough support.